hotel-reservation-system/src/components/Room/RoomTable.js

```diff
@@ -218,7 +282,7 @@ export default function RoomTable({ hotelId }) {
         size="large"
         margin="normal"
         className={classes.createRoomButton}
-        onClick={OpenAddRoomDialog}
+        onClick={() => OpenAddRoomDialog()}
```

Comment:
Why do you change this?

Reply:
because openAddRoomDialog has 1 parameter. without () => func(), the button event is sent to the parameter